spec/system/admin/reports_spec.rb

```diff
@@ -31,37 +31,54 @@
     end
   end
 
-  describe "Customers report" do
+  shared_examples "Can access Customers reports and generate report" do |inverse_columns_logic|
     before do
-      login_as_admin_and_visit admin_reports_path
+      allow(OpenFoodNetwork::FeatureToggle).to receive(:enabled?).with(
+        :report_inverse_columns_logic, anything
+      ).and_return(inverse_columns_logic)
     end
```

Comment:
Would it make sense to toggle this feature and let the build run to check for regressions?

If I understand correctly, currently only "Customers report" is tested with the feature toggled.

Reply:
Also `sales tax report`, but yes it's probably incomplete ...